tests/test_wps_subset_WFS.py

```diff
@@ -0,0 +1,309 @@
+import os
+import sys
+import unittest
+import configparser
+import json
+
+from pywps import Service
+from pywps.tests import client_for
+import numpy.ma as ma
+import netCDF4
+
+try:
+    from tests import test_wps_utils
+except ImportError:
```

Comment:
why is the `try` needed?

Reply:
@huard That's an Ouranos process. I leave it to you to change it correctly